### PR TITLE
cyclow implementation of twoway demo

### PR DESCRIPTION
**Pull request by ghost**
This pull request is tested on new jscomp

### CODE REVIEW

File: demos/twoway/cyclow/es6/src/phonebooth.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import {Block} from 'cyclow'
+
+const Phonebooth = () => Block({
+  on: {
+    'in.init': () => state => ({name: 'Steve'}),
+    'dom.text': text => state => ({name: text}),
+    'dom.click': () => ({
+      state: () => ({name: 'Batman'}),
+      'dom.action': el => el.firstElementChild.focus()
```
Comment (krawaller) [inline demos/twoway/cyclow/es6/src/phonebooth.js:9]: So the parameter `el` here will be the root element of the component?

Comment (ghost) [reply demos/twoway/cyclow/es6/src/phonebooth.js:9]: yes!

Comment (krawaller) [reply demos/twoway/cyclow/es6/src/phonebooth.js:9]: Neat! And I had never seen `firstElementChild` before, learning about that was an extra bonus for me :)

**Comment by krawaller** [comment]
Beautiful! I think this solution for the focusing is much better than the last one. Having focus as a local bool variable doesn't really map out very well, since it is really a piece of state belonging in the browser, and not on the individual element. 

But here you also solve the bigger problem of giving access to the node for whatever purpose. Neat!

**Pull request merged**